### DISCUSSION TITLE
GitHub account can be banned for contributing to a project used by illegal parties

### DISCUSSION THREAD

**Discussion by Coeur** [transmission/transmission#3618] [General]
Seems a bit tragic and against 1st Amendment (https://twitter.com/kurtopsahl/status/1556727681474957312), but:
1. Someone misused an open-source app to steal money
2. U.S. Department of the Treasury banned the app
3. GitHub banned a contributor of this open-source project

https://www.cryptotimes.io/github-disables-accounts-tied-to-tornado-cash-after-sanctions/

I hope 100% of Transmission users are using it legally, otherwise GitHub could kick us arbitrarily 😓.
I just wonder what would happen if someone misuses GitHub for illegal activities: will GitHub ban whoever contributed to GitHub?
And what if someone misuses a Microsoft Windows computer for illegal activities: will Microsoft ban whoever contributed to Windows?

**Comment by Petrprogs** [comment]
I can't imagine how you can use the transmission illegally... Although, probably even in the most harmless program it is possible to embed something bad.

**Comment by haffizzcoperate-ship-it** [comment]
why TikTok banned treasury.gov.my

**Comment by haffizzcoperate-ship-it** [comment]
babi
